deployment/updates/scripts/2016_04_05_migrate_tvinci_distribution_tags_to_be_dynamic.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+$realrun = false;
+
+if($argc != 2)
+{
+	echo "Arguments missing.\n\n";
+	echo "Usage: php 2016_04_05_migrate_tvinci_distribution_tags_to_be_dynamic.php {partner id}\n";
+	exit;
+}
+require_once (__DIR__ . '/../../bootstrap.php');
+$partnerId = $argv[1];
+
+
+$c = new Criteria();
+$c->add(DistributionProfilePeer::PARTNER_ID, $partnerId);
+$c->add(DistributionProfilePeer::STATUS, DistributionProfileStatus::ENABLED);
+$c->add(DistributionProfilePeer::PROVIDER_TYPE, TvinciDistributionPlugin::getDistributionProviderTypeCoreValue(TvinciDistributionProviderType::TVINCI));
+$distributionProfiles = DistributionProfilePeer::doSelect($c, $con);
+
+foreach ($distributionProfiles as $distributionProfile)
+{
+	/**
+	 * @var TvinciDistributionProfile $distributionProfile
+	 */
+	$tags = array();
+	$ismFileName = $distributionProfile->getFromCustomData('ismFileName', null, '');
+	$ismPpvmodule = $distributionProfile->getFromCustomData('ismPpvModule', null, '');
+	if ( $ismFileName != '' || $ismPpvmodule != '' )
+	{
+		$tag = createTvinciTag("ism", $ismFileName, $ismPpvmodule);
+		$tags[] = $tag;
+	}
+
+	$ipadnewFileName = $distributionProfile->getFromCustomData('ipadnewFileName', null, '');
+	$ipadnewPpvmodule = $distributionProfile->getFromCustomData('ipadnewPpvModule', null, '');
+	if ( $ipadnewFileName != '' || $ipadnewPpvmodule != '' )
+	{
+		$tag = createTvinciTag("ipadnew", $ipadnewFileName, $ipadnewPpvmodule);
+		$tags[] = $tag;
+	}
+
+	$iphonenewFileName = $distributionProfile->getFromCustomData('iphonenewFileName', null, '');
+	$iphonenewPpvmodule = $distributionProfile->getFromCustomData('iphonenewPpvModule', null, '');
+	if ( $iphonenewFileName != '' || $iphonenewPpvmodule != '' )
+	{
+		$tag = createTvinciTag("iphonenew", $iphonenewFileName, $iphonenewPpvmodule);
+		$tags[] = $tag;
+	}
+
+	$mbrFileName = $distributionProfile->getFromCustomData('mbrFileName', null, '');
+	$mbrPpvmodule = $distributionProfile->getFromCustomData('mbrPpvModule', null, '');
+	if ( $mbrFileName != '' || $mbrPpvmodule != '' )
+	{
+		$tag = createTvinciTag("mbr", $mbrFileName, $mbrPpvmodule);
+		$tags[] = $tag;
+	}
+
+	$dashFileName = $distributionProfile->getFromCustomData('dashFileName', null, '');
+	$dashPpvmodule = $distributionProfile->getFromCustomData('dashPpvModule', null, '');
+	if ( $dashFileName != '' || $dashPpvmodule != '' )
+	{
+		$tag = createTvinciTag("dash", $dashFileName, $dashPpvmodule);
+		$tags[] = $tag;
+	}
+
+	$widevineFileName = $distributionProfile->getFromCustomData('widevineFileName', null, '');
+	$widevinePpvmodule = $distributionProfile->getFromCustomData('widevinePpvModule', null, '');
+	if ( $widevineFileName != '' || $widevinePpvmodule != '' )
+	{
+		$tag = createTvinciTag("widevine", $widevineFileName, $widevinePpvmodule);
+		$tags[] = $tag;
+	}
+
+	$widevine_mbrFileName = $distributionProfile->getFromCustomData('widevineMbrFileName', null, '');
```

Comment:
widevine_mbrFileName Vs. widevineMbrFileName - I think widevineMbrFileName is better :)